data_driven_onboarding/data-driven-onboarding.sql

```diff
@@ -208,15 +208,24 @@ select
     -- will not return null, will return 0
     coalesce(R.unitsstab2007,0) as stabilized_unit_count_2007,
 
-    -- number of stabilized units at entered bbl in 2007
+    -- number of stabilized units at entered bbl in 2017
     -- drawn from rentstab_summary
     -- will not return null, will return 0
     coalesce(R.unitsstab2017,0) as stabilized_unit_count_2017,
```

Comment:
@toolness do we still need this in our SQL query now that the field is deprecated in our ddo schema?

Reply:
Aye we do, we should file a separate issue for removing the deprecated field eventually.  You will also want to add a `deprecation_reason` keyword argument to the GraphQL field for the 2017 value, to ensure that our front-end stops asking for it... see here for an example: https://github.com/JustFixNYC/tenants2/pull/1631/files#diff-8346124954d70e5043f4165d1ebfacccR127-R141

Reply:
Wait, isn't what that I did here? https://github.com/JustFixNYC/tenants2/pull/1690/files#diff-119127ee5f22a1d87751587413e310ddR171

Reply:
oh hahaha sneaky edit of that comment there ;)

Reply:
haha it is, i tried responding to this convo after I found out but Github wouldn't let me for some reason, so i edited my original reply thinking maybe it was part of my pending code review but it wasn't, and now this thread is completely confusing to anyone who reads it a year from now! lol